src/cli/cmd/pay.ts

```diff
@@ -80,6 +80,16 @@ export const pay = async (commandOptions) => {
     process.exit(0);
   }
 
+  if (!commandOptions.force) {
```

Comment:
I am usually all in for `--force` but in case of SW which handles money I do not think we should have this option. It too much simplifies possibility for double payouts. One should really put some effort if he wants to rerun existing payments - renaming old payout files. 

Technically we can enable this for networks other than mainnet if we can filter out. But in general I do not think it is good idea to have `--force`.